doc/gettingstarted.md

```diff
@@ -221,6 +221,22 @@ $ ttn-lw-cli end-devices create app1 dev2 \
 
 >Hint: You can also pass `--with-session` to have a session generated.
 
+It is also possible to create a multicast device (an ABP device which can not send uplinks and shares the security session with other devices) using the `--multicast` flag as follows:
+
+```bash
+$ ttn-lw-cli end-devices create app1 dev3 \
+  --frequency-plan-id EU_863_870 \
+  --lorawan-version 1.0.2 \
+  --lorawan-phy-version 1.0.2-b \
+  --abp \
```

Comment:
This should not be necessary @htdvisser?

Reply:
If `--abp` is not specified, `supports_join` is set to true. See [this](https://github.com/TheThingsNetwork/lorawan-stack/blob/861fe5eaec412b8da0ff47cdb431b41e49a30b50/cmd/ttn-lw-cli/commands/end_devices.go#L328-L367).
We could also check if its either `--abp` or `--multicast`.

Reply:
Can you file an issue about it?

Reply:
#889